helm-charts/sep-service/values.yaml

```diff
@@ -0,0 +1,94 @@
+fullName: anchor-platform
+namespace: sandbox
+service:
+   containerPort: 8080
+   servicePort: 8080
+   replicas: 1
+   type: NodePort
```

Comment:
NodePort might not work in all deployments. In fact if you use ingress ClusterIP is better.